.github/workflows/main.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       fail-fast: false
       matrix:
         # windows is tested on azure for now
-        os: [ubuntu-20.04, windows-2019, macos-10.15]
```

Comment:
What about using `[ubuntu-latest, windows-latest, macos-latest]`? Or is there a good reason to pick the oldest possible version?

Reply:
Mostly because that is what is used in the [cibuildwheel docs](https://cibuildwheel.readthedocs.io/en/stable/setup/#github-actions) and I had issues before deviating from that. Wheels on Mac and Windows are built without containers, so I could imagine there is an issue if the wheel is built on a newer version and is run on an older OS version, but the other direction usually works okay. So maybe that is why they try to build on older versions.